addons/mass_mailing/models/mailing.py

```diff
@@ -525,6 +527,51 @@ def _get_remaining_recipients(self):
         done_res_ids = [record['res_id'] for record in already_mailed]
         return [rid for rid in res_ids if rid not in done_res_ids]
 
+    def _send_statistics_mails(self):
+        """Send an email to the responsible of each finished mailing with the statistics."""
+        self.kpi_mail_required = False
+
+        for mailing in self:
+            user = mailing.user_id
```

Comment:
Maybe a mailing = mailing.with_context(lang=user.lang or self._context.get('lang')) could be interesting to have the body translated in the responsible language, not the one running the method (aka robodoo for crons).

cc @mart-e am I right with this one ?

Reply:
+1